public/app/plugins/panel/stat/types.ts

```diff
@@ -120,19 +120,24 @@ export function addOrientationOption<T extends SingleStatBaseOptions>(
   });
 }
 
-export function addTextSizeOptions<T extends SingleStatBaseOptions>(builder: PanelOptionsEditorBuilder<T>) {
-  builder.addNumberInput({
-    path: 'text.titleSize',
-    category: ['Text size'],
-    name: 'Title',
-    settings: {
-      placeholder: 'Auto',
-      integer: false,
-      min: 1,
-      max: 200,
-    },
-    defaultValue: undefined,
-  });
+export function addTextSizeOptions<T extends SingleStatBaseOptions>(
```

Comment:
is this still used? (it looks the same from commonOptionsBuilder)

Reply:
nope, left by mistake